charts/internal/shoot-cert-management-seed/templates/configmap-observability.yaml

```diff
@@ -30,6 +30,23 @@ data:
         regex: ^(cert_management_.+)$
         action: keep
 
+  alerting_rules: |
+    cert-controller-manager.rules.yaml: |
+      groups:
+      - name: cert-controller-manager.rules
+        rules:
+        - alert: SslCertificateWillExpireSoon
+          expr:  (cert_management_cert_object_expire-time())/86400 <= 3
+          for: 30m
+          labels:
+            service: cert-controller-manager
+            severity: critical
+            type: seed
+            visibility: all
+          annotations:
+            description: Certificate in namespace {{ .Release.Namespace }} will be expired in 3 days.
+            summary: SSL certificate will expire in 3 days
+
```

Comment:
Should the threshold be configurable? I fear that 3 days is fairly short, especially if you consider weekends and/or holidays.

I suppose this could be configured via the extension configuration. Then each cluster owner can use what fits their operational model.

WDYT?

Reply:
I have the same concern. 3 days are much too short. As the certificate should automatically be renewed 30 days before expiring, even it is expiring let's say in 15 days, something is going wrong.

Reply:
Hi @ScheererJ , @MartinWeindel ,
I will move the days before expiration to a configuration value in values.yaml.  
According to other extensions, such as Cloud Controller Manager, the alerting rules configured as configmap in [extension](https://github.com/gardener/gardener-extension-provider-aws/blob/cd0492cdb2eee347e8fc75435511f014b174e765/charts/internal/seed-controlplane/charts/cloud-controller-manager/templates/configmap-observability.yaml#L9).  So I think the Cert Controller Manager related monitoring should be configured here.

Reply:
That approach is certainly better. However, I was thinking about making it configurable for stakeholders as well. This would allow risk-averse operators to use a higher value while yolo-operators could disable it altogether.

@MartinWeindel / @cathyzhang05 WDYT?

Reply:
I will add code to make it configurable in the extension config.